crates/telio-utils/src/utils.rs

```diff
@@ -1,39 +1,61 @@
 /// Utilities
 ///
 
+#[macro_export]
+macro_rules! telio_log_generic {
+    ( $log_func: ident, $msg: expr) => {tracing::$log_func!($msg)};
+    ( $log_func: ident, $format: expr, $($arg:expr),+) => {{
+        use std::net::{IpAddr, SocketAddr};
+
+        #[allow(dead_code)]
+        trait HideSensitiveInfo {
+            fn hide_me_maybe(&self) -> &'static str {
+                "******"
+            }
+        }
```

Comment:
This needs to distinguish debug from release builds - only in release mode should the values be hidden.